src/openloco/widget.cpp

```diff
@@ -786,6 +786,7 @@ namespace openloco::ui::widget
 
         if (w->is_disabled(index))
         {
+            widget->left = 0x8000;
```

Comment:
I changed this so that the widget->left could be set to 0x8000. Which is then checked in drawTabs to see if the widget is disabled. But this causes the CI checks to fail due to if (widget.left != -2) in WIndowManager.cpp being always true. I will revert it and check for disabled widgets instead.